integration/test_run_scripts/command_line/all_tests-pe-centos6.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+SCRIPT_PATH=$(pwd)
+BASENAME_CMD="basename ${SCRIPT_PATH}"
+SCRIPT_BASE_PATH=`eval ${BASENAME_CMD}`
+
+if [ $SCRIPT_BASE_PATH = "command_line" ]; then
+  cd ../../
+fi
+
+export pe_dist_dir=http://pe-releases.puppetlabs.lan/3.7.1/
```

Comment:
The r10k package has been integrated with PE 3.8 so we need to test against 3.8 instead of 3.7. Change the address to "http://neptune.puppetlabs.lan/3.8/ci-ready/".